code/game/machinery/doors/airlock.dm

```diff
@@ -1370,7 +1370,7 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/Uncross(atom/movable/mover)
 	if(locate(/obj/effect/unwall_field) in loc) //Annoying workaround for this, especially because of that thing below -kanef
 		return 1
-	if(density && ismob(mover) && !(mover.checkpass(PASSGLASS) && !opacity) && !(mover.checkpass(PASSDOOR)) && !(istype(mover,/mob/living/simple_animal/shade)))//REEEEEEE
+	if(density && ismob(mover) && !(mover.checkpass(PASSGLASS) && !opacity) && !(mover.checkpass(PASSDOOR)) && !(istype(mover,/mob/living/simple_animal/shade)) && !(istype(mover,/mob/living/simple_animal/hostile/pulse_demon)))//REEEEEEE
```

Comment:
given that pulse demons only(?) move through wires, is there any reason not to simply give them the PASSDOOR flag?

Reply:
There are wires under windows for the shocked grilles, so it's probably for that.

Reply:
Yeah and some wires like the telecomms or AI ones go through walls, at least on Box.

Reply:
Then give them PASSGLASS and PASSWALL too instead of exceptions like this?

Reply:
There is no passwall flag.

Reply:
There is no passwall flag, gave them forceMove() when detecting a new cable instead.